src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java

```diff
@@ -98,4 +100,21 @@ public void shouldNotLogLongLinks() throws Exception {
         verify(checkConfig, getPath("InputLineLengthLongLink.java"), expected);
     }
 
+    @Test
+    public void countUnicodePointsOnce() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(LineLengthCheck.class);
+        checkConfig.addAttribute("max", "100");
+        // we need to set charset to let test pass on windows where default charset is not UTF-8
```

Comment:
not only windows, every system with other default encoding

Reply:
Fixed.